### ISSUE TITLE
linter fail in CI for github actions 

### ISSUE THREAD

**Issue by teacherSsamko** [cookiecutter/cookiecutter-django#3529]
## What happened?

linter in CI for github actions failed.
with 
```bash
No file in /home/runner/work/_backend/_backend matched to [**/requirements.txt], make sure you have checked out the target repository
```
while linter `set up python`

## Additional details

* Host system configuration:
    * Version of cookiecutter CLI (get it with `cookiecutter --version`): 1.7.3
    * OS name and version: ubuntu-latest

      On Linux, run
      ```bash
      lsb_release -a 2> /dev/null || cat /etc/redhat-release 2> /dev/null || cat /etc/*-release 2> /dev/null || cat /etc/issue 2> /dev/null
      ```

      On MacOs, run
      ```bash
      sw_vers
      ```

      On Windows, via CMD, run
      ```
      systeminfo | findstr /B /C:"OS Name" /C:"OS Version"
      ```

      ```bash
      # Insert here the OS name and version

      ```

    * Python version, run `python3 -V`: python 3.9
    * Docker version (if using Docker), run `docker --version`:
    * docker-compose version (if using Docker), run `docker-compose --version`:
    * ...
* Options selected and/or [replay file](https://cookiecutter.readthedocs.io/en/latest/advanced/replay.html):
  On Linux and MacOS: `cat ${HOME}/.cookiecutter_replay/cookiecutter-django.json`
  (Please, take care to remove sensitive information)
    ```json
    {
  "cookiecutter": {
    "project_name": "cookie_maria_sample",
    "project_slug": "cookie_maria_sample",
    "description": "Behold My Awesome Project!",
    "author_name": "Eunsub Lee",
    "domain_name": "comroom.net",
    "email": "ssamko@kakao.com",
    "version": "0.1.0",
    "open_source_license": "MIT",
    "timezone": "UTC",
    "windows": "n",
    "use_pycharm": "n",
    "use_docker": "y",
    "postgresql_version": "14.1",
    "js_task_runner": "None",
    "cloud_provider": "AWS",
    "mail_service": "Amazon SES",
    "use_async": "y",
    "use_drf": "y",
    "custom_bootstrap_compilation": "n",
    "use_compressor": "n",
    "use_celery": "y",
    "use_mailhog": "n",
    "use_sentry": "n",
    "use_whitenoise": "n",
    "use_heroku": "n",
    "ci_tool": "Github",
    "keep_local_envs_in_vcs": "y",
    "debug": "n",
    "_template": "https://github.com/cookiecutter/cookiecutter-django"
  }
}
    ```
<summary>
Logs:
<details>
<pre>
$ cookiecutter https://github.com/cookiecutter/cookiecutter-django
project_name [Project Name]: ...
</pre>
</details>
</summary>

**Comment by browniebroke** [comment]
Thanks for the report, but I think that's already been reported #3517 and should have been fixed in #3520 

Was the project generated before the 7th of January?

**Comment by teacherSsamko** [comment]
yes it was generated before the 7th of January. 
I searched before I wrote this issue. but I didn't know that #3520 is already there.
Thank you for your quick answer 👍
